### PR TITLE
final changes for v0.5

### PR DESCRIPTION
**Pull request by dtebbs**
- [x] removed redundant return
- [x] updated zeth submodule to latest

### CODE REVIEW

File: aggregator_server/aggregator_server.cpp [hunk 1]
```diff
@@ -81,7 +81,7 @@ static void load_keypair(
         keypair_file.c_str(), std::ios_base::in | std::ios_base::binary);
     in.exceptions(
         std::ios_base::eofbit | std::ios_base::badbit | std::ios_base::failbit);
-    return wsnark::keypair_read_bytes(keypair, in);
+    wsnark::keypair_read_bytes(keypair, in);
```
Comment (AntoineRondelet) [inline aggregator_server/aggregator_server.cpp:84]: LGTM, this return isn't needed.
We could stick to the approach used in Zeth though and change the returned type to a `snark:keypair` and declare the keypair in the function. However, we would have some copy of the keypair when return.
TL;DR is: this change looks good to me but we have some slight inconsistency between similar functions in Zeth and Zecale, so that would be valuable to clear that out (and probably move these functions in a namespace like `libzeth::server` that could then be used on zecale as `libzeth::server::load_keypair`)

(see: https://github.com/clearmatics/zeth/blob/develop/prover_server/prover_server.cpp#L44-L54 for ref)

Comment (AntoineRondelet) [reply aggregator_server/aggregator_server.cpp:84]: This is not strictly a blocker for the release, but better code factorization will mean, less code to maintain and less inconsistencies.

**Review by AntoineRondelet — APPROVED** [review]
LGTM, outstanding comment on code factorization across Zeth and Zecale moved in issue https://github.com/clearmatics/zeth/issues/417
